### ISSUE TITLE
Slowmotion in some games, choppy sound

### ISSUE THREAD

**Issue by Domingster** [hrydgard/ppsspp#18854]
### Game or games this happens in

Crash Mind over Mutant

### What area of the game

Everywhere in game I have enough fps (50) to run smoothly but game looks like is in slowmotion. I dont know why. In some games I have for example 25 fps but running normally, no slow motion. Souns is choppy. I dont know whats that. Can you help me?

### Speed seen in PPSSPP

75-80%, 50/60

### Platform

libretro / Retroarch

### Mobile device model or graphics card (GPU)

Anbernic RG353VS

### PPSSPP version affected

1.17.1

### Last working version

No workingversion

### Graphics backend (3D API)

OpenGL / GLES

### Checklist

- [X] Test in the [latest git build](https://buildbot.orphis.net/ppsspp/) in case it's already fixed.
- [X] [Search for other reports](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same issue.
- [ ] Try resetting settings or older versions and include if the issue is related.
- [X] Try changing graphics settings to determine if one causes the slowdown.
- [ ] Include logs or screenshots of issue.

**Comment by anr2me** [comment]
If it doesn't run at 100% there will be a chance for the sound to be choppy, even a couple of frames difference could have a noticeable cut-off on people who have sensitive hearing and familiar with how the sound should be.

**Comment by Domingster** [comment]
> If it doesn't run at 100% there will be a chance for the sound to be choppy, even a couple of frames difference could have a noticeable cut-off on people who have sensitive hearing and familiar with how the sound should be.
> 
> Regarding the slow motion, may be you can create a video of it? Was it something like the slowness in this issue #18509 ?

Yes, it looks quite same! What can I do? I see some option to lock 30 fps, any ideas how to do it?

**Comment by anr2me** [comment]
> > If it doesn't run at 100% there will be a chance for the sound to be choppy, even a couple of frames difference could have a noticeable cut-off on people who have sensitive hearing and familiar with how the sound should be.
> > Regarding the slow motion, may be you can create a video of it? Was it something like the slowness in this issue #18509 ?
> 
> Yes, it looks quite same! What can I do? I see some option to lock 30 fps, any ideas how to do it?

You can try underclocking it (ie. change the CPU clock to something between 50 and 200).

If you still couldn't get near 100% performance, you can try using frame skipping to reduce sound choppy-ness.

**Comment by Domingster** [comment]
> > > If it doesn't run at 100% there will be a chance for the sound to be choppy, even a couple of frames difference could have a noticeable cut-off on people who have sensitive hearing and familiar with how the sound should be.
> > > Regarding the slow motion, may be you can create a video of it? Was it something like the slowness in this issue #18509 ?
> > 
> > 
> > Yes, it looks quite same! What can I do? I see some option to lock 30 fps, any ideas how to do it?
> 
> You can try underclocking it (ie. change the CPU clock to something between 50 and 200).
> 
> If you still couldn't get near 100% performance, you can try using frame skipping to reduce sound choppy-ness.

Thank you, I will try. Could you explain how it can help, to underclock? I am not sure how clocking works but I thought I should try overclock to get better fps, that is not true?

**Comment by anr2me** [comment]
> > > > If it doesn't run at 100% there will be a chance for the sound to be choppy, even a couple of frames difference could have a noticeable cut-off on people who have sensitive hearing and familiar with how the sound should be.
> > > > Regarding the slow motion, may be you can create a video of it? Was it something like the slowness in this issue #18509 ?
> > > 
> > > 
> > > Yes, it looks quite same! What can I do? I see some option to lock 30 fps, any ideas how to do it?
> > 
> > 
> > You can try underclocking it (ie. change the CPU clock to something between 50 and 200).
> > If you still couldn't get near 100% performance, you can try using frame skipping to reduce sound choppy-ness.
> 
> Thank you, I will try. Could you explain how it can help, to underclock? I am not sure how clocking works but I thought I should try overclock to get better fps, that is not true?

This is only true on a real hardware, on an emulator it can counter-productive.

On games that tried to render as many frames as possible, overclocking can gives more burden (ie. attempts to render more frames), meanwhile underclocking may reduce the burden, thus can get a better/stable performance, especially if the game is doing GPU readbacks a lot, which is pretty expensive on performance.

**Comment by Domingster** [comment]
> > > > > If it doesn't run at 100% there will be a chance for the sound to be choppy, even a couple of frames difference could have a noticeable cut-off on people who have sensitive hearing and familiar with how the sound should be.
> > > > > Regarding the slow motion, may be you can create a video of it? Was it something like the slowness in this issue #18509 ?
> > > > 
> > > > 
> > > > Yes, it looks quite same! What can I do? I see some option to lock 30 fps, any ideas how to do it?
> > > 
> > > 
> > > You can try underclocking it (ie. change the CPU clock to something between 50 and 200).
> > > If you still couldn't get near 100% performance, you can try using frame skipping to reduce sound choppy-ness.
> > 
> > 
> > Thank you, I will try. Could you explain how it can help, to underclock? I am not sure how clocking works but I thought I should try overclock to get better fps, that is not true?
> 
> This is only true on a real hardware, on an emulator it can counter-productive.
> 
> On games that tried to render as many frames as possible, overclocking can gives more burden (ie. attempts to render more frames), meanwhile underclocking may reduce the burden, thus can get a better/stable performance, especially if the game is doing GPU readbacks a lot, which is pretty expensive on performance.
> 
> But the slowness issue, could also be a bug on timings, causing the game to misbehave.

So it did not help but thank you for explanation!

**Comment by AaronJk12378** [comment]
Is there a 30 fps hack for it?
